internal/pkg/inject/offset_results.json

```diff
@@ -448,6 +451,9 @@
                       "1.0.2",
                       "1.0.3",
                       "1.0.4",
+                      "1.0.5",
```

Comment:
Something seems off here, this version of the `google.golang.org/grpc/internal/transport` package does not exist as far as I can tell.

I'm also not able to find any version prior to [`v1.14.0`](https://pkg.go.dev/google.golang.org/grpc@v1.14.0/internal/transport?tab=versions)

Reply:
I can remove this for now, originally added it because of the offsets test